deploy/Chart/templates/database/statefulset.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: database-sts
```

Comment:
nit: I'd prefer not to encoded types in names. We keep it simple everywhere else.

Reply:
Is there value in being able to quickly discern what something is by its name?

Reply:
I personally don't think so. Everything in K8s is typed. You never see `Deployments` and `StatefulSets` in a list next to each other without another column showing the types.